frontend/config-assistant/src/components/toolbar/TopToolbar.vue

```diff
@@ -243,6 +267,12 @@ watch(storeToRefs(useSessionStore()).fileData, () => {
 const searchTerm: Ref<string> = ref('');
 const searchBarVisible = ref(false);
 
+const initialSchemaSelectionDialog = ref();
+// Function to show the category selection dialog
+const showCategoryDialog = () => {
```

Comment:
I would only quickly update the name of this function to `showInitialSchemaDialog` then everything is very nice